numpy/lib/_function_base_impl.py

```diff
@@ -4726,18 +4728,24 @@ def _quantile(
     # Virtual because it is a floating point value, not an valid index.
     # The nearest neighbours are used for interpolation
     try:
-        method = _QuantileMethods[method]
+        method_props = _QuantileMethods[method]
     except KeyError:
         raise ValueError(
             f"{method!r} is not a valid method. Use one of: "
             f"{_QuantileMethods.keys()}") from None
-    virtual_indexes = method["get_virtual_index"](values_count, quantiles)
+    virtual_indexes = method_props["get_virtual_index"](values_count,
+                                                        quantiles)
     virtual_indexes = np.asanyarray(virtual_indexes)
 
     supports_nans = (
             np.issubdtype(arr.dtype, np.inexact) or arr.dtype.kind in 'Mm')
 
-    if np.issubdtype(virtual_indexes.dtype, np.integer):
+    int_virtual_indices = np.issubdtype(virtual_indexes.dtype, np.integer)
+    supports_integers = method_props["fix_gamma"] is None
+    supports_integers = supports_integers or \
```

Comment:
This PR needs a rebase unfortunately.  Just commenting here to nitpick that I would prefer no `\` continuation :).

Reply:
Did a rebase (with force push). The `\` was there to avoid a line too long, I rewrote the style to make it fit.

Reply:
Thanks, the other solution (with the exception of `with` statements) is to just use parantheses `(...)` which allow spanning multiple lines :).